internal/util-complete/src/main/scala/sbt/internal/util/complete/Parser.scala

```diff
@@ -383,8 +387,8 @@ trait ParserMain {
   }
 
   /** Presents a Char range as a Parser.  A single Char is parsed only if it is in the given range.*/
-  implicit def range(r: collection.immutable.NumericRange[Char]): Parser[Char] =
-    charClass(r contains _).examples(r.map(_.toString): _*)
+  implicit def range(r: collection.immutable.NumericRange[Char], label: String): Parser[Char] =
```

Comment:
I just noticed bincompat breakages in this PR now. Even though Parser is currently under "internal" package, it's publicly used so I don't think we can break bincompat.